torch/nn/functional.py

```diff
@@ -3683,26 +3683,41 @@ def huber_loss(
     target: Tensor,
     reduction: str = "mean",
     delta: float = 1.0,
+    weights: Optional[Tensor] = None,
```

Comment:
Please name these `weight` rather than `weights` for consistency with other loss functions with this argument in this file, similarly in the other functions